src/index.html

```diff
@@ -6,17 +6,402 @@
+              href="#index"
+            >
+              <img
+                class="products__photo"
+                src="./images/Image-product4.png"
+                alt="image-product4"
+              />
+            </a>
+          </div>
+
+          <h3 class="products__items">Audio</h3>
+        </article>
+
+        <article class="products__container">
+          <div class="products__images">
+            <a
+              class="products__image products__image--wide"
+              href="#index"
+            >
+              <img
+                class="products__photo"
+                src="./images/Image-product5.png"
+                alt="image-product5"
+              />
+            </a>
+
+            <a
+              class="products__image products__image--small"
+              href="#index"
+            >
+              <img
+                class="products__photo"
+                src="./images/Image-product6.png"
+                alt="image-product6"
+              />
+            </a>
+          </div>
+
+          <h3 class="products__items">Accessories</h3>
+        </article>
+
+        <button class="products__button">View all</button>
+      </section>
+
+      <section class="about main__section">
+        <div class="about__image-wrap">
+          <img
+            class="about__image"
+            src="./images/Image-layout.png"
+            alt="image-layout"
+          />
+        </div>
+
+        <div class="about__content">
+          <h2 class="about__tittle">About us</h2>
+          <p class="about__text">
+            We’re a London-based tech company on a mission to remove barriers
+            between people and technology.
+            <br />
+            <br />
+            To do this, we’re crafting intuitive, flawlessly connected products
+            that improve our lives without getting in the way.
+            <br />
+            No confusing tech-speak. No silly product names. Just artistry,
+            passion and trust. And products we’re proud to share with our
+            friends and family. Simple.
+            <br />
+            <br />
+            Because beautiful tech shouldn’t be complicated.
+          </p>
+        </div>
+      </section>
+    </main>
+
+    <footer class="footer">
+      <h2 class="footer__tittle">Contact us</h2>
+
+      <div class="footer__grid">
+        <div class="contact footer__section">
+          <form
+            class="contact__form"
+            action="index.html"
+            method="post"
+            onsubmit="this.reset(); return false;"
+          >
+            <div class="contact__field">
+              <input
+                required
+                placeholder="Name"
+                class="contact__input"
+                type="text"
+              />
+            </div>
+
+            <div class="contact__field">
+              <input
+                required
+                placeholder="Email"
```

Comment:
Placeholder should be a valid value for this input [Reference](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md#:~:text=Placeholders%20should%20give%20examples%20of%20what%20should%20be%20added%2C%20to%20make%20the%20expected%20format%20clear)